client/utils/reduxFormUtils.js

```diff
@@ -44,7 +44,11 @@ export function validateSettings(formProps) {
   if (formProps.currentPassword && !formProps.newPassword) {
     errors.newPassword = 'Please enter a new password or leave the current password empty.';
   }
-
+  if (formProps.newPassword) {
+    if (formProps.newPassword < 6) {
```

Comment:
i think you could combine this into one if statement, and, you're missing .length here:
```js
if (formProps.newPassword && formProps.newPassword.length < 6) {
// ...
}
```

Reply:
@catarak I have done the changes, Please review :)